### PR TITLE
Implemented Deep Learning Algorithms

### LINKED ISSUE #50: Implement deep learning algorithm to improve accuracy
### Prerequisites

- [X] I am an Open Source Contributor
- [X] I checked to make sure that this issue has not already been filed

### Description

Most of the time deep learning techniques are better than ML techniques. Maybe trying out a deep learning technique would be better on the dataset. I plan to try Convolution Neural Network(CNN)

### Screenshots

_No response_

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/ML-Project-Drug-Review-Dataset/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.

### PR DESCRIPTION
**Pull request by SRINIVAS-2003**
# Fixes Issue🛠

Closes #50 

# Description👨‍💻 

I have implemented ANN and AdaBoost algorithms and plotted some graphs like accuracy plot , predicted VS testing and training data values for the above two algorithms. A confusion matrix of the above two algorithms is also performed.

# Type of change📄

- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)

# How this has been tested✅

I have tested accuracy using ANN and AdaBoost algorithms.
Mainly I have used Google tool (Google Colab) to run and test the project whether the code is working or not.

# Checklist✅ 

- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have added demonstration in the form of GIF/video file
- [] I am an Open Source Contributor

# Screenshots/GIF📷

### CODE REVIEW

**Comment by SRINIVAS-2003** [comment]
@Rakesh9100 This time I haven't faced any issue and I completed all the tasks which u have said. The code is also linted now.
Please once verify it.
I hope it will work this time.
Very sorry for troubling u this much but this time I have raised it properly with the name [Implenetation of DL Algorithms](https://github.com/Rakesh9100/ML-Project-Drug-Review-Dataset/pull/118/commits/5db937980708d95304456256850a8f6a88247892). 
Really Sorry for taking taking your time @Rakesh9100

**Comment by Rakesh9100** [comment]
No need to apologize for reaching out or taking up my time. I'm here to assist you, and it's my pleasure to help @SRINIVAS-2003

**Comment by Rakesh9100** [comment]
LGTM!! Congrats on getting your PR merged to this repo 🎉🎉 @SRINIVAS-2003

**Comment by SRINIVAS-2003** [comment]
> LGTM!! Congrats on getting your PR merged to this repo 🎉🎉 @SRINIVAS-2003 

Thank you very much @Rakesh9100
